client/js/socket-events/msg.js

```diff
@@ -35,7 +35,8 @@ socket.on("msg", function(data) {
 		.trigger("msg", [
 			target,
 			data
-		]);
+		])
+		.trigger("keepToBottom");
```

Comment:
Okay, this goes along with the rename from msg to keepToBottom and I missed that.